src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/set_interior_vehicle_data_request_test.cc

```diff
@@ -186,38 +188,36 @@ TEST_F(SetInteriorVehicleDataRequestTest,
 TEST_F(
     SetInteriorVehicleDataRequestTest,
     Execute_ValidWithSettableAndReadOnlyParams_ExpectCutReadOnlyAndResendToHMI) {
-  // Arrange
```

Comment:
@mvorobio What is the reason to remove such comments?

Reply:
@AByzhynar These comments are useless enough, because a pattern they introduce is sometimes difficult to follow. E.g. expectations from the ASSERT section could be placed around the ACT section.
In other words, such comments rarely give useful info but add unnecessary lines of code and sometimes even confuse.
But it's my personal opinion only. I wouldn't argue if these comments are required by some reasons.

Reply:
@mvorobio Please return the back. It is the full responsibility of the developer to put the code to the right section.

Reply:
@AByzhynar please check https://github.com/mvorobio/sdl_core/pull/2/commits/794597b6469711802915fb788ab9bb4fec9ae004